src/language_server/custom_requests/ShowAst.mli

```diff
@@ -0,0 +1,2 @@
+val meth : string
+val on_request : Jsonrpc.Structured.t option -> Yojson.Safe.t option
```

Comment:
I'm actually not sure of the best way to do this, because I'm not sure that we could make each file conform to a signature, for example. We could put a module in each file that ascribes to the same signature, but that would encapsulate the data twice, and also I figure each `on_request` method might have different parameters. For now, I'm OK with different signatures for each file.

Reply:
Makes sense, that's what I was thinking to but was unsure